languages.toml

```
@@ -2076,14 +2076,14 @@ scope = "source.gleam"
 injection-regex = "gleam"
 file-types = ["gleam"]
 roots = ["gleam.toml"]
-comment-token = "//"
+comment-tokens = ["//", "///", "////"]
 indent = { tab-width = 2, unit = "  " }
 language-servers = [ "gleam" ]
 auto-format = true
 
 [[grammar]]
 name = "gleam"
-source = { git = "https://github.com/gleam-lang/tree-sitter-gleam", rev = "6ece453acf8b14568c10f629f8cd25d3dde3794f" }
+source = { git = "https://github.com/gleam-lang/tree-sitter-gleam", rev = "0d8b4b595daf76ed30c65ab22e01504fa64a1eac" }
```

```suggestion
source = { git = "https://github.com/gleam-lang/tree-sitter-gleam", rev = "ee93c639dc82148d716919df336ad612fd33538e" }
```

I pushed some changes recently to update the tree-sitter version. Now the grammar should be on the latest tree-sitter ABI